Source/WebKit/WebProcess/WebPage/WebFrame.messages.in

```diff
@@ -22,4 +22,6 @@
 
 messages -> WebFrame {
     ContinueWillSubmitForm(WebKit::FormSubmitListenerIdentifier listenerID)
+    DidCommitLoadInAnotherProcess()
+    DidFinishLoadInAnotherProcess()
```

Comment:
In the future, it seems we'll need IPC to indicate that the provisional load failed in another process, and that the load failed in another process.

Reply:
There's already a FIXME comment for this exact thing.  So many tests to add.  So many things to get working!